### PR TITLE
configure feature URL for dynamic image & log level by environment

### LINKED ISSUE #31: Allow to specify internal repo mirror for Dynamic images

### PR DESCRIPTION
**Pull request by htreu**
Closes #31

### CODE REVIEW

File: dynamic/org.ops4j.pax.url.mvn.cfg.tmpl [hunk 1]
```diff
@@ -92,4 +92,4 @@ org.ops4j.pax.url.mvn.defaultRepositories=file:${karaf.home}/${karaf.default.rep
 #    @noreleases : the repository does not contain any released artifacts
 #    @id=repository.id : the id for the repository, just like in the settings.xml this is optional but recommended
 #
-org.ops4j.pax.url.mvn.repositories=https://artifact-public.instana.io/artifactory/features-public@id=features@snapshots@snapshotsUpdate=always,https://artifact-public.instana.io/artifactory/shared@id=shared@snapshots@snapshotsUpdate=always
+org.ops4j.pax.url.mvn.repositories={{ getenv "INSTANA_FEATURE_MVN_URL" }},https://artifact-public.instana.io/artifactory/shared@id=shared@snapshots@snapshotsUpdate=always
```
Comment (mmanciop) [inline dynamic/org.ops4j.pax.url.mvn.cfg.tmpl]: What happens if this is undefined? Shouldn't we have an `if` around it?

Comment (htreu) [reply dynamic/org.ops4j.pax.url.mvn.cfg.tmpl]: it gets set in run.sh if undefined from env.

**Comment by mmanciop** [comment]
@htreu we merge this? :-)

File: dynamic/org.ops4j.pax.logging.cfg.tmpl [hunk 2]
```diff
@@ -26,7 +26,7 @@
 
 # Configuration of the Instana logger. Changes will be hot-reloaded.
 log4j2.logger.instana.name = com.instana
-log4j2.logger.instana.level = INFO
+log4j2.logger.instana.level = {{ getenv "INSTANA_LOG_LEVEL" }}
```
Comment (mmanciop) [inline dynamic/org.ops4j.pax.logging.cfg.tmpl]: I think we should do some validation here. It is bad if logging borks :-)

**Pull request merged**